src/app.py

```diff
@@ -1122,7 +1145,16 @@ def publish_datastage(identifier):
                 ds_path = ingest_helper.dataset_directory_absolute_path_published(data_access_level,
                                                                                   dataset_group_uuid, dataset_uuid)
                 md_file = os.path.join(ds_path, "metadata.json")
-                json_object = entity_json_dumps(entity_instance, dataset_uuid)
+                try:
+                    prov_metadata_url = f"{app.config['ENTITY_WEBSERVICE_URL']}/datasets/{dataset_uuid}/prov-metadata"
```

Comment:
We should use `commons_file_helper.removeTrailingSlashURL()` on the base URL in case the configuration may include a trailing slash, which had happened in the past.

Reply:
done